### DISCUSSION TITLE
How to Disable SSL for HTTP Sender Connector in Mirth Connect?

### DISCUSSION THREAD

**Discussion by PonnumaniGurusamy** [nextgenhealthcare/connect#6300] [Q&A] [answered]
I’m trying to make an API call using the HTTP Sender connector in Mirth Connect, but I'm getting the following error:

Error: QUEUED: Error connecting to HTTP server [SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target]

The same API call works fine when using Postman. I suspect that the issue might be related to SSL, and I’m wondering if there’s a way to disable SSL in Mirth Connect for this connector.

Could anyone guide me on how to disable SSL in Mirth Connect or provide any alternative solutions to bypass this issue?

Thanks in advance!

**Comment by pacmano1** [comment]
There are many similar posts that answer your question.

if you're calling a TLS endpoint, you can't disable it. The other posts describe how to add the certificate authority to the required key store.

Also check out the mirth forums site where this question has been asked as well.

**Comment by PonnumaniGurusamy** [comment]
Thank you for the answer. It is not a TLS endpoint. Could you please guide me on how to disable it?

**Reply by pacmano1** [reply]
What is the endpoint URL?

**Reply by pacmano1** [reply]
If you are calling anything with https:// it is TLS.

Are you calling something like mirth's own API, it doesn't run over http.  You may also be calling an http endpoint that is redirecting you to https.   You need to provide more information.

**Reply by PonnumaniGurusamy** [reply]
It is outside HTTPS endpoint

![image](https://github.com/user-attachments/assets/2cb8b9b1-11d7-4c06-92ff-423facca20c7)

![image](https://github.com/user-attachments/assets/9b597922-1a08-4d26-a601-7ac7d343e2e0)

**Reply by pacmano1** [reply, marked as answer]
You have https.  That means it is TLS.   

You need to add the CA to the java key store if the TLS certificate signer is not trusted.  Like I said, this has been asked and answered before.

**Reply by PonnumaniGurusamy** [reply]
I tired multiple ways. But all are failing. Is any manual available  for adding the CA to java keystore?
